File: 23andme_processor.py
f=open("test_genome.txt")

total_same = 0
total_different = 0

for i, line in enumerate(f):
    if line[0]=="#":
        continue
    else:
        d = line.strip().split("\t")

        if d[1] == "MT":
            continue
        same = d[3][0] == d[3][1]
        
        if same:
            total_same += 1
        else:
            total_different += 1

print total_same
print total_different

File: 23andme_processor.rb
f = IO.readlines("test_genome.txt")

total_same = 0
total_different = 0

f.each do |line|

	next if line[0] == "#"
		data = line.chomp().split("\t")

		next if data[1] == "MT"

			same = data[3][0] == data[3][1]

			if same
				total_same += 1
			else
				total_different += 1
			end
	end

	puts total_same
	puts total_different